docs/medical-api/more-info/pagination.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: "Pagination"
+description: "Retrieving larger amounts of data."
+---
+
+Some of Metriport's endpoints are paginated: they return a limited amount of items each time they are requested. 
+Each request being a single "page".
+
+In order to have access to all items, after we receive the items for the current page we can request the
+next page to the API.
+
+Each response includes:
+- `meta`: the information about the response.
+- `<items>`: the list of items for the respective request, where `items` is named according to the respective
+  type of data being requested - e.g., `patients`, `documents`.
+
+And `meta` contains:
+- `itemsOnPage`: the amount of items on the current page.
+- `nextPage`: (optional) the URL to return the next page; if not present it means there's not a next page
+  and all items have been returned.
+- `prevPage`: (optional) the URL to return the previous page; if not present it means we're on the first page.
+
+## Pagination parameters
+
+Metriport's pagination is designed simplify its usage, hiding away the complexity for most situations.
```

Comment:
nit: Metriport's pagination is designed {(to) <- i think this is missing} simplify its usage

Reply:
done